### PR TITLE
Denormalized recents

### PR DESCRIPTION
**Pull request by edpaget**
I'll rebase this after #820

### CODE REVIEW

**Comment by camallen** [comment]
@edpaget can you rebase this please?

**Comment by edpaget** [comment]
It's rebased.

File: db/migrate/20150507212315_create_recents.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateRecents < ActiveRecord::Migration
+  def change
```
Comment (camallen) [inline db/migrate/20150507212315_create_recents.rb]: Should we just ignore any prev recents for staging and prod?

**Comment by camallen** [comment]
Looks good.

File: db/migrate/20150507212315_create_recents.rb (hunk 1 above)
Comment (edpaget) [inline db/migrate/20150507212315_create_recents.rb]: I added one. It will take ages to run it against production.

**Pull request merged**